### PR TITLE
Feat: localnet subcommand

### LINKED ISSUE #1: Add command to run local node from zksync-cli
Add the ability to run a local node from zksync-cli

We should integrate [matter-labs/local-setup](https://github.com/matter-labs/local-setup) with these two new commands:

`zksync-cli node run `
`zksync-cli node stop `
`zksync-cli node logs `
`zksync-cli node help`

(should we have more commands?)

We should display some developer-friendly messages with each command

### PR DESCRIPTION
**Pull request by jpcenteno**
closes #1 #2

### CODE REVIEW

**Comment by jpcenteno** [comment]
```

        _      ____                             _                          _                  _   
  ____ | | __ / ___|   _   _   _ __     ___    | |   ___     ___    __ _  | |  _ __     ___  | |_ 
 |_  / | |/ / \___ \  | | | | | '_ \   / __|   | |  / _ \   / __|  / _` | | | | '_ \   / _ \ | __|
  / /  |   <   ___) | | |_| | | | | | | (__    | | | (_) | | (__  | (_| | | | | | | | |  __/ | |_ 
 /___| |_|\_\ |____/   \__, | |_| |_|  \___|   |_|  \___/   \___|  \__,_| |_| |_| |_|  \___|  \__|
                       |___/                                                                      
USAGE: zksync-cli localnet <operation>

Manage local L1 and L2 chains

Available operations
  start   -- Start L1 and L2 localnets
  down    -- Stop L1 and L2 localnets
  clear   -- Reset the localnet state
  logs    -- Display logs
  help    -- Display this message and quit
  wallets -- Display seeded wallet keys
```

File: src/localnet.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+const runCommand = (command: string) => {
+  try {
+    // runs given command and prints its output to console
+    execSync(`${command}`, { stdio: 'inherit' });
+  } catch (error) {
+    console.error('Failed to run command: ', error);
+    return false;
+  }
+  return true;
+};
```
Comment (ilitteri) [inline src/localnet.ts:18]: I saw that we have the same function in [src/create.ts](https://github.com/lambdaclass/zksync-cli/blob/main/src/create.ts#L4-L17). Would it be a good idea to abstract it?

**Review by ilitteri — COMMENTED** [review]
LGTM

File: .github/workflows/secrets_scanner.yaml
Comment (mationorato) [inline .github/workflows/secrets_scanner.yaml:1]: what is this @jpcenteno ?

**Review by mationorato — COMMENTED** [review]
LGTM, but looks at the comments.

File: .github/workflows/secrets_scanner.yaml
Comment (jpcenteno) [reply .github/workflows/secrets_scanner.yaml:1]: I don't know

Comment (jpcenteno) [reply .github/workflows/secrets_scanner.yaml:1]: I don't understand why I appear as a co-author of that commit.

**Review by mationorato — APPROVED** [review]
LGTM

**Pull request merged**